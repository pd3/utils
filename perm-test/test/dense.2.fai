1	56
2	56
3	56
4	56
5	56
