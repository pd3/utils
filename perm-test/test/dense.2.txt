
perm-test -t test/dense.2.tgt.txt -b test/dense.2.bg.txt -f test/dense.2.fai -c test/dense.2.calls.txt -n 1e9

             10        20        30        40        50
    12345678901234567890123456789012345678901234567890123456
       bbb bbbb bbb           bbb              bbbb      bbb

    12345678901234567890123456789012345678901234567890123456
       bbb bbbb bbb           bbb              bbbb      bbb

    12345678901234567890123456789012345678901234567890123456
       bbb bbbb bbb           bbb              bbbb      bbb

    12345678901234567890123456789012345678901234567890123456
       bbb bbbb bbb           bbb              bbbb      bbb

    12345678901234567890123456789012345678901234567890123456
       bbb bbbb bbb     ttt   bbb   ttt ttt    bbbb tttt bbb
                    ccccccc
    ....... ....... ....ooo ....... ooo.ooo ....... xxxx...
     ....... ....... ...ooo. ....... oo.ooo. ....... xxx....
      ....... ....... ..ooo.. ......x ..ooo.. ......x
       ....... ....... .ooo... .....xx .ooo... .....xx
        ....... ....... xxx.... ....xx. ooo.... ....xxx
         ....... ....... xx..... ...ooo. xx..... ...xxxx
          ....... ......x x...... ..ooo.o x...... ..xxxx.
           ....... .....oo ....... .ooo.oo ....... .xxxx..

    exact: 30./(5*50) = 0.12
    perm-test:          0.120015051
    err:   1.5e-05

perm-test -t test/dense.2.tgt.txt -b test/dense.2.bg.txt -f test/dense.2.fai -c test/dense.2.calls.txt -n 1e9 --no-bg-overlap

    exact: 13./(5*50) = 0.052
    perm-test:      0.052001216
    err:   1.216e-06


