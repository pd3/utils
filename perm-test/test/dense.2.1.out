BG	1	4	6
BG	1	8	11
BG	1	13	15
TGT	1	21	23
BG	1	27	29
TGT	1	33	35
TGT	1	37	39
BG	1	44	47
TGT	1	49	52
BG	1	54	56
BG	2	4	6
BG	2	8	11
BG	2	13	15
BG	2	27	29
BG	2	44	47
BG	2	54	56
BG	3	4	6
BG	3	8	11
BG	3	13	15
BG	3	27	29
BG	3	44	47
BG	3	54	56
BG	4	4	6
BG	4	8	11
BG	4	13	15
BG	4	27	29
BG	4	44	47
BG	4	54	56
BG	5	4	6
BG	5	8	11
BG	5	13	15
BG	5	27	29
BG	5	44	47
BG	5	54	56
MSG	Maximum chromosome randomization error due to length discretization: 1.5e-03%
CALL	TGT	1	17	23
NCALLS	1	0
